optuna/samplers/_cmaes.py

```diff
@@ -272,17 +279,12 @@ def __init__(
         self._cma_rng = LazyRandomState(seed)
         self._search_space = IntersectionSearchSpace()
         self._consider_pruned_trials = consider_pruned_trials
-        self._restart_strategy = restart_strategy
         self._initial_popsize = popsize
```

Comment:
The population size is fixed (here or in the `sample_relative`). How about renaming this `self._popsize`?